### PR TITLE
feat(MessageType): add missing types

### PR DESCRIPTION
**Pull request by almeidx**
**Please describe the changes this PR makes and why it should be merged:**

**If applicable, please reference Discord API Docs PRs or commits that influenced this PR:**
- discord/discord-api-docs#5815
- discord/discord-api-docs#5826

### CODE REVIEW

File: payloads/v10/channel.ts [hunk 1]
```diff
@@ -688,6 +689,14 @@ export enum MessageType {
 	GuildInviteReminder,
 	ContextMenuCommand,
 	AutoModerationAction,
+
+	InteractionPremiumUpsell = 26,
```
Comment (kyranet) [inline payloads/v10/channel.ts:693]: Shouldn't we have `RoleSubscriptionPurchase` (25) above this one? That way we don't need to specify it. 👀 

I don't see any open PR adding that type.

Comment (almeidx) [reply payloads/v10/channel.ts:693]: #665

**Review by Jiralite — CHANGES_REQUESTED** [review]
Some message types were reverted in https://github.com/discord/discord-api-docs/pull/5826. We should probably keep them, but use the `@unstable` tag, I guess? @vladfrangu

File: payloads/v10/channel.ts [hunk 2]
```diff
@@ -688,6 +689,14 @@ export enum MessageType {
...
+	StageStart,
+	StageEnd,
+	StageSpeaker,
+	StageRaiseHand,
+	StageTopic,
```
Comment (kyranet) [inline payloads/v10/channel.ts:698]: Those need to be removed, ref: https://github.com/discord/discord-api-docs/pull/5826

**Review by kyranet — CHANGES_REQUESTED** [review]

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**